src/cloud-api-adaptor/docker/README.md

```diff
@@ -161,3 +218,7 @@ For debugging you can use docker commands like `docker ps`, `docker logs`, `dock
 kubectl delete deployment nginx
 ```
 
+## Troubleshooting
+
+When using `containerd` and `nydus-snapshotter` you might encounter pod creation failure due to
+issues with unpacking of image. Check the `nydus-snapshotter` troubleshooting [doc](../docs/troubleshooting/nydus-snapshotter.md).
```

Comment:
It is worth giving and example command of exec-ing into the default kind cluster e.g. `docker exec -it $(docker ps -aqf name=peer-pods-worker) /bin/bash` command for lazy people like me?

Reply:
Hmm, you can also do `kubectl debug node/peer-pods-worker -it --image=busybox` to login to the cluster node as well.
And adding the docker command to the nydus-snapshotter.md doesn't make sense as the the troubleshooting doc is generic.
Let me just add the command to README

Reply:
Yeah - the debug is a nicer more generic way to do it. Thanks!